ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -367,12 +366,22 @@ protected IngestionResult ingestFromStreamImpl(StreamSourceInfo streamSourceInfo
 
     @Override
     protected String getClientType() {
-        return MANAGED_STREAMING_INGEST_CLIENT;
+        return CLASS_NAME;
```

Comment:
Can't the parent class define this, without the need to `override`, since it's always going to return `CLASS_NAME``?

Reply:
You can't inherit static types, so it will have to be non-static, and the string will have to be created every time.
Since it uses reflection for the class name, it's wasteful to do every time, so I'm leaving it as is.